.github/workflows/sonatypeStaticDeployment.yml

```diff
@@ -0,0 +1,48 @@
+name: Sonatype Deployment
+
+on:
+  pull_request:
+    branches: ["main", "release/*"]
+  workflow_dispatch:
+    inputs:
+      agent:
+        description: 'NewRelic Static Android agent deployment'
+        default: '6.+'
+        required: false
+
+jobs:
+  assemble_project:
+    name: Assemble project
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Setup JDK 11
+      uses: actions/setup-java@v3
+      with:
+          java-version: '11'
+          distribution: 'temurin'
+          cache: 'gradle'
+    - name: Build with Gradle
+      run: ./gradlew clean assemble --no-daemon -PmonoEnabled=true
```

Comment:
Is it a good idea using if statement to combine these two sonatype deployment since monoEnabled=true is the only difference?

Reply:
we can do that, let me do change for that